AudioFile.h

```diff
@@ -1318,7 +1318,13 @@ int AudioFile<T>::getIndexOfChunk (std::vector<uint8_t>& source, const std::stri
         if ((i + 4) >= source.size())
             return -1;
         
-        auto chunkSize = fourBytesToInt (source, i, endianness);
+        uint32_t chunkSize = fourBytesToInt (source, i, endianness);
```

Comment:
This all looks great thank you - but I think for this line it should be a `int32_t` as that is what is returned by the `fourBytesToInt()` function

Reply:
If you can make that change then I can run the tests and it should be all good to merge :)

Reply:
Great! I've updated my branch.

Reply:
Great, thank you - will merge now :)